### PR TITLE
chore: update project name references

### PR DESCRIPTION
**Pull request by lucasrod16**

### CODE REVIEW

**Comment by phillebaba** [comment]
I think there is some overlap between this and #2735, lets get that merged before this.

File: src/pkg/transform/git_test.go [hunk 1]
```diff
@@ -48,23 +48,23 @@ func TestMutateGitURLsInText(t *testing.T) {
...
 	expectedText := `
 	# Here we handle invalid URLs (see below comment)
 	# We transform https://*/*.git URLs
-	https://gitlab.com/repo-owner/zarf-1211668992.git
+	https://gitlab.com/repo-owner/zarf-4156197301.git
 	# Even URLs with things on either side
-	stuff https://gitlab.com/repo-owner/zarf-1211668992.git andthings
+	stuff https://gitlab.com/repo-owner/zarf-4156197301.git andthings
 	# Including ssh://*/*.git URLs
-	https://gitlab.com/repo-owner/zarf-2566185087.git
```
Comment (phillebaba) [inline src/pkg/transform/git_test.go:67]: Why do these change?

**Review by phillebaba — COMMENTED** [review]
Small comment that needs explanation

File: src/pkg/transform/git_test.go [hunk 2]
```diff
@@ -119,29 +119,29 @@ func TestGitURLtoFolderName(t *testing.T) {
...
+		"zarf-agent-927663661",
 		"some-cool-repo-1916670310",
 		"podinfo-1350532569",
 		"podinfo-1853010387",
 		"twistlock-1920149257",
 		"podinfo-122075437",
 		"zarf-public-test-612413317",
 		"zarf-public-test-634307705",
-		"zarf.helm-2570741950",
-		"zarf-2175050463",
```
Comment (phillebaba) [inline src/pkg/transform/git_test.go:133]: Ditto

File: src/pkg/transform/git_test.go (hunk 1 above)
Comment (lucasrod16) [reply src/pkg/transform/git_test.go:67]: `MutateGitURLsInText` appends a crc32 hash of the repo name to the end of the URL. The repo name changed so the hash changed

https://github.com/zarf-dev/zarf/blob/4bd52447194a771d38f6694027d90b6ee8c4aa2b/src/pkg/transform/git.go#L81-L86

Comment (phillebaba) [reply src/pkg/transform/git_test.go:67]: Makes sense in that case

**Review by phillebaba — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
 
-[![Latest Release](https://img.shields.io/github/v/release/defenseunicorns/zarf)](https://github.com/defenseunicorns/zarf/releases)
-[![Go version](https://img.shields.io/github/go-mod/go-version/defenseunicorns/zarf?filename=go.mod)](https://go.dev/)
-[![Build Status](https://img.shields.io/github/actions/workflow/status/defenseunicorns/zarf/release.yml)](https://github.com/defenseunicorns/zarf/actions/workflows/release.yml)
-[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/defenseunicorns/zarf/badge)](https://securityscorecards.dev/viewer/?uri=github.com/defenseunicorns/zarf)
+[![Latest Release](https://img.shields.io/github/v/release/zarf-dev/zarf)](https://github.com/zarf-dev/zarf/releases)
+[![Go version](https://img.shields.io/github/go-mod/go-version/zarf-dev/zarf?filename=go.mod)](https://go.dev/)
+[![Build Status](https://img.shields.io/github/actions/workflow/status/zarf-dev/zarf/release.yml)](https://github.com/zarf-dev/zarf/actions/workflows/release.yml)
+[![Zarf Documentation Status](https://api.netlify.com/api/v1/badges/fe846ae4-25fb-4274-9968-90782640ee9f/deploy-status)](https://app.netlify.com/sites/zarf-docs/deploys)
```
Comment (phillebaba) [inline README.md:6]: Please remove

File: README.md [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/zarf-dev/zarf/badge)](https://securityscorecards.dev/viewer/?uri=github.com/zarf-dev/zarf)
 
 <img align="right" alt="zarf logo" src="site/src/assets/zarf-logo.png"  height="256" />
 
 [![Zarf Website](https://img.shields.io/badge/web-zarf.dev-6d87c3)](https://zarf.dev/)
 [![Zarf Documentation](https://img.shields.io/badge/docs-docs.zarf.dev-775ba1)](https://docs.zarf.dev/)
 [![Zarf Slack Channel](https://img.shields.io/badge/k8s%20slack-zarf-40a3dd)](https://kubernetes.slack.com/archives/C03B6BJAUJ3)
-[![Community Meetups](https://img.shields.io/badge/community-meetups-22aebb)](https://github.com/defenseunicorns/zarf/issues/2202)
+[![Community Meetups](https://img.shields.io/badge/community-meetups-22aebb)](https://github.com/zarf-dev/zarf/issues/2202)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)
```
Comment (phillebaba) [inline README.md:15]: Please remove

File: README.md (hunk 3 above)
Comment (lucasrod16) [reply README.md]: removed

File: README.md (hunk 4 above)
Comment (lucasrod16) [reply README.md]: removed

**Review by phillebaba — APPROVED** [review]

**Pull request merged**